backend-implicaction/src/test/java/com/dynonuggets/refonteimplicaction/controller/JobsPostingControllerTest.java

```diff
@@ -126,4 +131,63 @@ void getJobByIdWithoutJwtShouldBeForbidden() throws Exception {
 
         verify(jobPostingService, times(0)).getJobById(anyLong());
     }
+
+    @Test
+    @WithMockUser
+    void archiveJobShouldChangeStatus() throws Exception {
+        JobPostingDto jobPostingDtoAvailable = JobPostingDto.builder()
+                .id(1L)
+                .createdAt(Instant.now())
+                .description("description")
+                .location("Paris")
+                .keywords("toto,yoyo")
+                .salary("1111$")
+                .status(statusAvailable)
+                .build();
+
+        JobPostingDto jobPostingDtoArchived = JobPostingDto.builder()
+                .id(1L)
+                .createdAt(Instant.now())
+                .description("description")
+                .location("Paris")
+                .keywords("toto,yoyo")
+                .salary("1111$")
+                .status(statusArchived)
+                .build();
```

Comment:
tu peux simplifier en ne créant qu'un seul dto et en ne valuant qu'un minimum de champs genre id et archive. 

```suggestion
        JobPostingDto expectingDto = JobPostingDto.builder()
                .id(1L)
                .archive(true)
                .build();
```